contracts/src/mcms/merkle-proof.ts

```diff
@@ -30,7 +30,7 @@ export function build(
   const opProofs: OpProofs = Array.from({ length: ops.length }, (_, i) => computeProof(i))
 
   const encodeProof = (v) => beginCell().storeUint(v, 256)
-  const encodeSignature = (v) => mcms.builder.data.signature.encode(v).asBuilder()
```

Comment:
As an observation, we've went from calling `asBuilder` 11 times in this PR to calling `asCell` 244 times :)